### PR TITLE
Pop up menu

### PR DESCRIPTION
**Pull request by EyasuTeshome**
We have included a pop-up menu. mainly we made changes to index.html, style.css, and index.js. Now the pop-up menu can be initiated by the "see projects" button and closed with the close icon on the right top corner of the popup menu.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -98,7 +102,7 @@ <h2>Tonic</h2>
                     </ul>
                 </div>
                 <div class="button">
-                    <button type="button" class="work-button">See Project</button>
+                    <button id="1" type="button" class="work-button see-project-btn">See Project</button>
```
Comment (tufoinnkuo10) [inline index.html:105]: - [ ] please kindly remove the underline from the `see live and see source` link.. also kindly shift the popup close icon inside to make it more visible
Screenshot (184)_LI
s

File: index.html [hunk 2]
```diff
@@ -130,7 +134,7 @@ <h2>Tonic</h2>
                     </ul>
                 </div>
                 <div class="button">
-                    <button type="button" class="work-button">See Project</button>
+                    <button id="2" type="button" class="work-button">See Project</button>
```
Comment (tufoinnkuo10) [inline index.html:137]: - [ ] your popup image is showing half. kindly fix that
Screenshot (184)

File: index.html [hunk 3]
```diff
@@ -160,7 +164,7 @@ <h2>Tonic</h2>
                     </ul>
                 </div>
                 <div class="button">
-                    <button type="button" class="work-button">See Project</button>
+                    <button id="3" type="button" class="work-button">See Project</button>
```
Comment (tufoinnkuo10) [inline index.html:167]: - [ ] kindly make the mobile version full height.
your version 👇🏾 
Screenshot (185)
figma 👇🏾 
Screenshot (186)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @EyasuTeshome 

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ popup looks ok
✅ array works good
✅ responsive

## Required Changes ♻️♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @EyasuTeshome 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**